src/ZODB/DB.py

```diff
@@ -363,6 +363,7 @@ def __init__(self,
                  databases=None,
                  xrefs=True,
                  large_record_size=1<<24,
+                 explicit_transactions = False,
```

Comment:
Sigh, I missed that. This is one of the places where PEP 8 (and Guildo) are wrong.